app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,22 @@
+module SessionsHelper
+  # Logs in the given user.
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  # Returns the current logged-in user (if any).
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
toán tử `||=` có tác dụng gì?

Reply:
nếu @current_user là nil hoặc undefined thì sẽ gán với User.find_by, nếu ko thì ko gán